handler/handler.go

```diff
@@ -39,7 +42,10 @@ func handle(request types.Request) {
 			Name: proto.String("Eremetic"),
 			User: proto.String(""),
 		},
-		Scheduler: scheduler,
+		Scheduler:        scheduler,
+		BindingAddress:   net.ParseIP("0.0.0.0"),
```

Comment:
Would this be the address in the config file? Which I realize might not even make sense that it's configurable.